src/content/configuration/dev-server.md

```diff
@@ -1353,6 +1353,113 @@ webpack-dev-server --stdin
 ```
 
 
+## `devServer.transportMode`
+
+`string = 'sockjs': 'sockjs' | 'ws'` `object`
+
+W> `transportMode` is an experimental option, meaning its usage could potentially change without warning.
+
+T> Providing a string to `devServer.transportMode` is a shortcut to setting both `devServer.transportMode.client` and `devServer.transportMode.server` to the given string value.
+
+This option allows to choose the current `devServer` transport mode for client/server individually or to provide custom client/server implementation. This allows to specify how browser or other client communicates with the `devServer`.
+
+The current default mode is [`'sockjs'`](https://www.npmjs.com/package/sockjs). This mode uses [SockJS-node](https://github.com/sockjs/sockjs-node) as a server, and [SockJS-client](https://www.npmjs.com/package/sockjs-client) on the client.
+
+Use `'sockjs'` mode:
```

Comment:
I just realized, we dont give examples of default behavior, can you please get rid of that?

Reply:
The rest looks good

Reply:
You meant remove the example, right?